### PR TITLE
Add ttf

### LINKED ISSUE #313: PDFs can't handle Unicode characters with built-in fonts
```
A Prawn::Errors::IncompatibleStringEncoding occurred in application_submissions#show:

  Your document includes text that's not compatible with the Windows-1252 character set.
If you need full UTF-8 support, use TTF fonts instead of PDF's built-in fonts.

  app/pdfs/print_record_pdf.rb:42:in `block in table_content'


-------------------------------
Request:
-------------------------------

  * URL        : https://transit-jobapps.admin.umass.edu/application_submissions/1933.pdf
  * HTTP Method: GET
  * IP address : 128.119.98.81
  * Parameters : {"controller"=>"application_submissions", "action"=>"show", "id"=>"1933", "format"=>"pdf"}
  * Timestamp  : 2017-11-06 07:16:27 -0500
  * Server : af-transit-app3.admin.umass.edu
  * Rails root : /srv/jobapps/releases/20170915134017
  * Process: 11273

-------------------------------
Backtrace:
-------------------------------

  app/pdfs/print_record_pdf.rb:42:in `block in table_content'
  app/pdfs/print_record_pdf.rb:41:in `table_content'
  app/pdfs/print_record_pdf.rb:11:in `initialize'
  app/controllers/application_submissions_controller.rb:97:in `new'
  app/controllers/application_submissions_controller.rb:97:in `block (2 levels) in show'
  app/controllers/application_submissions_controller.rb:94:in `show'
```

In this _particular_ case, the applicant's employer address had a `ZERO WIDTH SPACE` in it which could be stripped, but we probably should allow Unicode characters in applicant responses. Using a TrueType font with Prawn doesn't look too hard.

### PR DESCRIPTION
**Pull request by Anbranin**
Closes #313

### CODE REVIEW

**Comment by werebus** [comment]
She added the file, `app/assets/fonts/DejaVuSans.ttf`, which is a binary (non-text) file so you don't see a diff.

**Comment by Anbranin** [comment]
This happened again today. It'd be nice if it were approved, or if I need to make changes I could work on that.
I am wondering how to force code-climate to report that everything is fine, because it is: https://codeclimate.com/github/umts/jobapps/pull/319#issues

**Comment by werebus** [comment]
I don't think "codeclimate/coverage" works? I've run into issues elsewhere where it never returned (of course I can't remember where).

File: app/pdfs/print_record_pdf.rb [hunk 1]
```diff
@@ -49,7 +49,9 @@ def table_content(content_width, column_width, record)
         cells.padding = 12
         self.header = true
         self.column_widths = [column_width, column_width]
-        self.cell_style = { borders: [:bottom], border_width: 0.5, font: "DejaVu Sans"}
+        self.cell_style = {
```
Comment (avacmeyer) [inline app/pdfs/print_record_pdf.rb]: Does this space have to be here?

**Comment by Anbranin** [comment]
I've run into this before as well and I did something funky to fix it. I can't remember if that something worked. I can't read up on this right now because of my eyes (I'm touch-typing and relying on spell-check rn lol) so I'm going to say fk it right this min

**Pull request merged**